docs/web-console-guide/master.adoc

```diff
@@ -127,8 +127,11 @@ Assessing applications refers to estimating the risks and costs involved in prep
 
 Analyzing applications refers to using rules to determine which specific lines in an application must be modified before the application can be migrated or modernized.
 
+include::topics/mta-assessment-changes-700.adoc[leveloffset=+2]
 include::topics/mta-web-assessing-apps.adoc[leveloffset=+2]
-include::topics/mta-web-applying-assessments-to-other-apps.adoc[leveloffset=+3]
+include::topics/mta-web-assessment-module.adoc[leveloffset=+3]
+include::topics/mta-web-archetypes.adoc[leveloffset=+3]
+// include::topics/mta-web-applying-assessments-to-other-apps.adoc[leveloffset=+3]
```

Comment:
‘Copy assessment/review’ option is not relevant anymore and has been removed in MTA 7.0.0.